central/reports/service/v2/service_impl.go

```diff
@@ -328,10 +333,12 @@ func (s *serviceImpl) GetMyReportHistory(ctx context.Context, req *apiV2.GetRepo
 		return nil, errors.Wrap(errox.InvalidArgs, err.Error())
 	}
 
+	//Add request type ondemand, schduled to filter out view based reports
 	conjunctionQuery := search.ConjunctionQuery(
 		search.NewQueryBuilder().
 			AddExactMatches(search.ReportConfigID, req.GetId()).
-			AddExactMatches(search.UserID, slimUser.GetId()).ProtoQuery(),
+			AddExactMatches(search.UserID, slimUser.GetId()).
+			AddExactMatches(search.ReportRequestType, storage.ReportStatus_ON_DEMAND.String(), storage.ReportStatus_SCHEDULED.String()).ProtoQuery(),
```

Comment:
I find this pattern more readable than the one up top. Also, we should be more consistent. Pick one pattern and change for consistency,